src/components/pages/Chat/EmptyFeed/index.tsx

```diff
@@ -51,10 +52,14 @@ const UnstyledEmptyFeed = ({ className, roomCreatedAt }: Props) => {
                         {format(roomCreatedAt, 'iiii, LLL do yyyy')}
                     </CreatedAt>
                 )}
-                <AddMemberButton type="button">
-                    <img src={AddMemberIcon} alt="" />
-                    <span>Add member</span>
-                </AddMemberButton>
+                <AddMemberModal
+                    button={
+                        <AddMemberButton type="button">
+                            <img src={AddMemberIcon} alt="" />
+                            <span>Add member</span>
+                        </AddMemberButton>
+                    }
+                />
```

Comment:
We could simply throw that `AddMemberModal` into `AddMemberButton` and skip the entire `button={…}` operation.